pkg/gofr/datasource/file/ftp/interface.go

```diff
@@ -3,11 +3,82 @@ package ftp
 import (
 	"context"
 	"io"
+	"os"
 	"time"
 
 	"github.com/jlaffaye/ftp"
 )
 
+// FTPFile represents a file in the filesystem.
+type FTPFile interface {
```

Comment:
There is no need to name this FTPFile in ftp package


```suggestion
// File represents a file in the filesystem.
type File interface {
```

Especially when next interface is named FileInfo

Reply:
I completely agree with your point but we can't keep it just File as then it is colliding with the struct named `File` that has all the implementation of the interface.